src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(int firstSide, int secondSide, Color color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        super.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(

```

Comment:
You should implement the 'draw' method in a way to separate the logic of displaying the figure information from the actual output. This will make the code more testable and reusable.